mule-apikit-common/src/main/java/org/mule/apikit/common/APISyncUtils.java

```diff
@@ -47,4 +47,66 @@ public static String toApiSyncResource(String s) {
                                       EXCHANGE_TYPE, resourceParts[length - 1]);
     return apiSyncResource;
   }
+
+  public static boolean compareResources(String resourceA, String resourceB, Boolean checkVersion) {
+    APISyncResource apiSyncResourceA = null;
+    APISyncResource apiSyncResourceB = null;
+    try {
+      apiSyncResourceA = new APISyncResource(resourceA);
+      apiSyncResourceB = new APISyncResource(resourceB);
+    } catch (APISyncResourceException e) {
+      return false;
+    }
+
+    return apiSyncResourceA.equals(apiSyncResourceB, checkVersion);
+  }
+
+  private static class APISyncResource {
+
+    private String groupId;
+    private String artifact;
+    private String version;
+    private String classifier;
+    private String packager;
+    private String file;
+
+
+    APISyncResource(String resource) throws APISyncResourceException {
+      if (!APISyncUtils.isSyncProtocol(resource))
+        throw new APISyncResourceException("Invalid APISync Resource");
```

Comment:
I would not display the key APISync to the customers, it seems an internal name

Reply:
The exception is private to the APISyncUtils class